app/tests/components_tests/factories.py

```diff
@@ -21,4 +21,6 @@ class ComponentInterfaceValueFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = ComponentInterfaceValue
 
+    pk = factory.Sequence(lambda n: n + 9999999)
```

Comment:
Why do we need a lambda here?

Reply:
the civ is expected to have a pk here: https://github.com/comic/grand-challenge.org/blob/master/app/grandchallenge/components/models.py#L227. Apparently the factory hasn't set a pk at that point if it's not explicitly included here. I agree this isn't the nicest solution, but I couldn't think of a better option. I'll think it over again though.

Reply:
Weird, is that a problem in the codebase itself? Would be strange if we added files or values to a None folder.

Reply:
No I think it has to do with the way the factories are invoked. It errored hard, so we would have seen this happen in production for sure.